metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/UpdateIndicesHook.java

```diff
@@ -165,7 +166,7 @@ private void handleUpdateChangeEvent(@Nonnull final MetadataChangeLog event) {
         event.hasSystemMetadata() ? event.getSystemMetadata().getRunId() : null);
 
     // Step 2. For all aspects, attempt to update Graph
-    if (_diffMode && _graphService instanceof ElasticSearchGraphService) {
+    if (_diffMode && (_graphService instanceof ElasticSearchGraphService || _graphService instanceof Neo4jGraphService)) {
```

Comment:
since we only have elastic search and neo4j graph services supported, I think this check can simply be `if(_diffMode) { ...`